docs/walkthrough.md

```diff
@@ -138,7 +138,16 @@ chmod +x ./kubectl
 We'll assume hereafter that all `kubectl` commands are using this
 newly-installed executable.
 
-To configure `kubectl` to communicate with the service catalog APU server:
+## Step 5 - Configuring `kubectl` to Talk to the API Server
+
+To configure `kubectl` to communicate with the service catalog API server, we'll have to
+get the IP address of the `Service` (and just the IP - no ports or `http://` prefixes)
+that sits in front of the API server pod(s). If you installed the catalog with one of the 
+`helm install` commands above, then this service will be called `catalog-catalog-apiserver`, 
```

Comment:
well, w/o something people will be stuck at this point - as I was until morgan helped me